src/main/kotlin/ro/hibyte/betting/dto/BetDTO.kt

```diff
@@ -0,0 +1,23 @@
+package ro.hibyte.betting.dto
+
+import ro.hibyte.betting.entity.Bet
+import ro.hibyte.betting.entity.UserProfile
+
+class BetDTO(
+    val betId: Long? = null,
+    var user: Long? = null,
+
+    //betType
```

Comment:
Remove this comment please (unless it's something important, then let me know)

Reply:
it is something that needed to be implemented in Alexandra's code and it was there when I merged with her from my understanding it was resolved in her code